### ISSUE TITLE
System.InvalidOperationException Crash

### ISSUE THREAD

**Issue by Zegura** [SpoinkyNL/Artemis#240]
```
System.InvalidOperationException

Message:
Collection was modified; enumeration operation may not execute.

Stack Trace:
   at System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   at System.Collections.Generic.List`1.Enumerator.MoveNextRare()
   at Artemis.Profiles.Layers.Types.KeyPress.KeyPressType.Draw(LayerModel layerModel, DrawingContext c)
   at Artemis.Profiles.ProfileModel.DrawLayers(Graphics g, IEnumerable`1 renderLayers, IDataModel dataModel, Rect rect, Boolean preview, Boolean updateAnimations, String updateType)
   at Artemis.Models.EffectModel.Render(RenderFrame frame, Boolean keyboardOnly)
   at Artemis.Managers.LoopManager.Render()
   at Artemis.Managers.LoopManager.LoopTimerOnElapsed(Object sender, ElapsedEventArgs elapsedEventArgs)

TargetSite:
Void ThrowInvalidOperationException(System.ExceptionResource)

Source:
mscorlib

HResult:
-2146233079
```

**Comment by SpoinkyNL** [comment]
Hey, resolved in 1.7, thanks for the report

**Comment by Zegura** [comment]
No worries. Thanks for making this program, it's amazing <3
